packages/devtools/cli-next/src/commands/index.ts

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright 2025 DXOS.org
+//
+
+import { Command, Options } from '@effect/cli';
+
+import { ENV_DX_PROFILE_DEFAULT } from '@dxos/client-protocol';
+
+import { ClientService, ConfigService } from '../services';
```

Comment:
i still prefer index files not to have anything except imports, but feel free to push back.

Reply:
yeah it started as just command which grouped the sub-commands but then during cleanup ended up being where the services were provided... can think about another way to organize the commands